extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -116,8 +117,6 @@ public static function onInstantGlobalsGetNewsAndStoriesVariables( array &$vars
 		// TODO: Remove after src=[gpt,ns] is finished
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app